pom.xml

```diff
@@ -39,7 +39,7 @@
     <properties>
         <java.version>11</java.version>
         <springfox.version>3.0.0-SNAPSHOT</springfox.version>
-        <reactor-rabbitmq-streams.version>0.0.8</reactor-rabbitmq-streams.version>
+        <reactor-rabbitmq-streams.version>0.0.7</reactor-rabbitmq-streams.version>
```

Comment:
oh not a downgrade... I was actually pointing to 0.0.7 explicitly in the pom and wasn't using this property so just updated to reflect correct version. I left it at 0.0.7 because it looks like we need to update reactor/spring boot since 0.0.8 uses functions that don't exist here